message_definitions/v1.0/messages.xml

```diff
@@ -774,6 +774,7 @@
       <field name="status" type="uint8" values="STOPPED|RUNNING|ERROR">General status of the logger</field>
       <field name="errno" type="uint8">Error number, depend of the logging system, provides detailed information</field>
       <field name="used" type="uint32" unit="bytes">Accumulated number of bytes written by the logger</field>
+      <field name="filenames" type="char[]"/>
```

Comment:
several files possible here ? please add a description.
also don't forget to update in paparazzi part the chibios logger that is using this message

Reply:
I will add the description. The update in paparazzi is on its way.